src/applications/yellow-ribbon/containers/SearchForm/index.jsx

```diff
@@ -185,21 +185,23 @@ export class SearchForm extends Component {
           />
         </div>
 
-        {/* Unlimited Contribution Amount */}
-        <ErrorableCheckbox
-          checked={contributionAmount === 'unlimited'}
-          label="Only show schools that fund all tuition and fees not covered by Post-9/11 GI Bill benefits"
-          onValueChange={onCheckboxChange('contributionAmount')}
-          required={false}
-        />
-
-        {/* Unlimited Number of Students */}
-        <ErrorableCheckbox
-          checked={numberOfStudents === 'unlimited'}
-          label="Only show schools that provide funding to all eligible students"
-          onValueChange={onCheckboxChange('numberOfStudents')}
-          required={false}
-        />
+        <div>
```

Comment:
I don't know what it does, but does this need `className="form-checkbox-buttons"`?

Reply:
That's what I see at https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/platform/user/profile/vet360/containers/ReceiveTextMessages.jsx#L96

Reply:
It doesn't, `form-checkbox-buttons` has no styles applied to it lol